examples/java/pom.xml

```diff
@@ -138,12 +137,17 @@
                   <goal>verify</goal>
                 </goals>
                 <configuration>
+                  <includes>
+                    <include>WordCountIT.java</include>
+                    <include>WindowedWordCountIT.java</include>
+                  </includes>
                   <parallel>all</parallel>
                   <threadCount>4</threadCount>
                   <systemPropertyVariables>
                     <beamTestPipelineOptions>
                       [
                       "--project=apache-beam-testing",
+                      "--tempLocation=gs://temp-storage-for-end-to-end-tests",
```

Comment:
It seems because we should really be using a Test version of the DirectRunner or it should also  honor the --tempRoot concept like all the other test runners. As you suggest, we should fix this in another PR.